### DISCUSSION TITLE
Error with Openssl

### DISCUSSION THREAD

**Discussion by LuizMateuss** [composer/composer#10993] [Q&A / Support]
I'm trying install composer, but apears this problem with openssl, i searshed on internet but found nothing usefull to solve this problem

![Screenshot_2022-08-15_01-13-04](https://user-images.githubusercontent.com/99288913/184577154-74b8db5d-0818-4617-a3cc-37a606d2f0b2.png)
![Screenshot_2022-08-15_01-18-02](https://user-images.githubusercontent.com/99288913/184577163-0f1478af-56ba-4959-9715-6b858d7ae0ea.png)

**Comment by Seldaek** [comment]
I guess you need to install the openssl package before you can build the openssl php extension.. But this is really more about installing PHP than a Composer question.
